Off-Top-App/off-top-java
# OT-90 | Receive audio data from the websocket

This pull request coincides with off-top-flutter OT-90 (https://github.com/Off-Top-App/off-top-flutter/issues).

**What I Did:**
- The simple idea behind this task is to safely receive the audio file in some form from flutter through our WebSocket.
- Note: The more ambitious idea here is to end up with the same audio file and play it as confirmation. As is, we are accepting the file in type ArrayList<Integer> byte form from flutter, but we are failing to properly process and write it to an audio file. That being said, there's a simple implementation of the write-in here that has been commented out so we can work on it in the future. 

**How To Test:**
- When the flutter and spring-boot server is running make way to the WebSocket page. Click on the "send data" button to see data coming in. You will see a bunch of numbers appearing, that is fine because it is byte data of the audio file.
- Check the code and see if there can be any improvements.
